bindings/wasm/src/lib.rs

```diff
@@ -26,13 +26,25 @@ pub mod crypto;
 pub mod did;
 pub mod error;
 pub mod iota;
-pub mod revocation;
-
+pub mod jose;
 pub mod resolver;
+pub mod revocation;
+pub mod storage;
 
 /// Initializes the console error panic hook for better error messages
 #[wasm_bindgen(start)]
 pub fn start() -> Result<(), JsValue> {
   console_error_panic_hook::set_once();
   Ok(())
 }
+
+// This section should be used as the central place for imports from `./lib`
+// used by TS definitions written on the Rust side.
+// It appears the import path must be relative to `src`.
+#[wasm_bindgen(typescript_custom_section)]
+const CUSTOM_IMPORTS: &'static str = r#"
+import { JwsAlgorithm } from '../lib/jose/jws_algorithm';
```

Comment:
I think we should be able to import everything from `../lib/jose` because of the index

Reply:
Even if we don't need it? Some enums are not required as imports, so I tend towards being specific here, but not strongy opposed.